JM/CF/Scripts/1_Core/CommunityFramework/Files/CF_File.c

```diff
@@ -320,6 +320,110 @@ class CF_File : Managed
 
 		return true;
 	}
+	
+	/**
+	 * @brief Opens file and returns all text in a single string
+	 * 
+	 * @param path The path to your file
+	 * 
+	 * @return all contents of the file in a single string
+	 */
+	static string ReadAllText(string path)
+	{
+		CF_File file = new CF_File(path);
+		if (!file.IsValid())
+		{
+			return string.Empty;
+		}
+		
+		string fileContents;
+		CF_FileStream fileStream = file.CreateStream(FileMode.READ);
+		CF_Byte byte = fileStream.Next();
+		while (byte) 
+		{
+			fileContents += byte.AsciiToString();
+			byte = fileStream.Next();
+		}
+		
+		fileStream.Close();
+		
+		return fileContents;
+	}	
```

Comment:
I think for performance reasons something like this should not be read through the whole file io pipeline with every single character going through multiple script calls? Wouln't `FGets` be an order of magnitude faster?

What was your use case when you came up with the idea to add these 3 methods?